src/EvoSC.Common/Config/Models/IDatabaseConfig.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.ComponentModel;
+using Config.Net;
+using Tomlet.Attributes;
+
+namespace EvoSC.Common.Config.Models;
+
+public interface IDatabaseConfig
+{
+    public enum DatabaseType
+    {
+        MySql
```

Comment:
I think the goal was to support more than just MySql, but I think it's fine for the beginning. Just something to keep in mind if we want to support MSSQL and Postgresql in the future.

Reply:
Yes, that's why I added the enum, so we can extend it in the future.